setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import find_packages
+from setuptools import setup
+from glob import glob
+from os.path import basename
+from os.path import splitext
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="falcon-integration-gateway",
+    version="0.0.1",
+    author="CrowdStrike",
+    maintainer="Simon Lukasik",
```

Comment:
In a future PR, feel free to put something like "CrowdStrike Solution Architecture" if you want. Not meant to take away your work, but rather help deflect any direct EMails you might get in the future ("your name is in the code! help!")